src/NServiceBus.Core/Transports/TransportOperation.cs

```diff
@@ -0,0 +1,36 @@
+namespace NServiceBus.Transports
+{
+    /// <summary>
+    /// Defines the transport operations including the message and information how to send it.
+    /// </summary>
+    public class TransportOperation
+    {
+        OutgoingMessage message;
+        DispatchOptions dispatchOptions;
```

Comment:
Using properties with private setters seems to be our standard and is less lines of code?